source/docs/zero-to-robot/step-1/intro-to-frc-robot-wiring.rst

```diff
@@ -278,14 +278,15 @@ Using the Dual Lock or hardware, fasten all components to the board. Note that i
 .. raw:: html
 
     <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-        <iframe src="https://www.youtube-nocookie.com/embed/Zsnro-9_ZfQ" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
+        <iframe src="https://www.youtube-nocookie.com/embed/SfkmyN9mYLA?si=dMHW1JCVkbpDjbu6&amp;start=39" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
     </div>
 
 
 A number of the CAN and power connectors in the system use a Weidmuller LSF series wire-to-board connector. There are a few things to keep in mind when using this connector for best results:
 
 - Wire should be 16 AWG (1.5 :math:`mm^2`) to 24 AWG (0.25 :math:`mm^2`) (consult rules to verify required gauge for power wiring)
 - Wire ends should be stripped approximately 5/16 (~8 mm)"
+- A ferrule can be added to reinforce the connection and prevent splaying.
```

Comment:
```suggestion
- A ferrule can be optionally added to reinforce the connection and prevent splaying, as shown in the video above
```

Reply:
I thought about adding optionally but isn't that redundant?  If I say something can be done it is naturally an option.

Reply:
Yes, but my worry is that someone just reading the steps will not realize that, so saying optionally makes it more obvious